app/internal/page/certificateprovider/enter_reference_number.go

```diff
@@ -14,30 +14,31 @@ import (
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
 )
 
-type enterReferenceCodeData struct {
+type enterReferenceNumberData struct {
```

Comment:
any reason you changed to reference number? ABCDEF123456 isn't really a number

Reply:
Just including here for future reference as I know we covered on slack:

> Content flagged they'd agreed to use it taking in to account things like NI number, Vehicle reg number and driving licence number all containing letters and gave a term that most people would understand (figma was updated yesterday to reflect this)